git-open.1.md

```diff
@@ -0,0 +1,138 @@
+# git-open - Open the repository's website in your browser.
+
+
+## SYNOPSIS
+
+`git-open` [remote-name] [branch-name]
```

Comment:
Remove the dash here, also maybe the quotes (not sure how your md -> man converter handles that). Let's change the names to match `git-push` (change to repository and refspec), also make "optional". I think it should be 1 line as well?

git open [--issue] [<repository>] [<refspec>]

Reply:
All done, but the back ticks stayed.